specification/sql/resource-manager/Microsoft.Sql/stable/2018-06-01/managedInstances.json

```diff
@@ -0,0 +1,217 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-06-01",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/sqlAgent/current": {
+      "get": {
+        "tags": [
+          "ManagedInstances"
+        ],
+        "description": "Gets current instance sql agent configuration.",
+        "operationId": "SqlAgent_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved current sql agent configuration.",
+            "schema": {
+              "$ref": "#/definitions/SqlAgent"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 SqlAgentManagedServerNotFound - User tried to configure sql agent of unknown managed instance."
+          }
+        },
+        "x-ms-examples": {
+          "Gets current instance sql agent configuration.": {
+            "$ref": "./examples/SqlAgentGet.json"
+          }
+        }
+      },
+      "put": {
```

Comment:
>put [](start = 7, length = 3)

I see that you don't have a DELETE call for this resource type. Is it because it always exists and can only be updated?

Reply:
Yes